src/guide/essentials/component-basics.md

```diff
@@ -466,7 +466,7 @@ Which might render something like:
 Something bad happened.
 :::
 
-This can be achieved using Vue's custom `<slot>` element:
+This can be achieved using Vue's custom `<slot>` element, which in our example is being used in the AlertBox component implementation:
```

Comment:
I agree with the sentiment of the code below being unclear. In the spirit of "show, don't tell", maybe we could remove the added explanation and add the comment in the code indicating it's alert box?

```html
<!-- AlertBox.vue -->
<template>
  <div class="alert-box">
    <strong>This is an Error for Demo Purposes</strong>
    <slot />
  </div>
</template>

Reply:
I agree with you, I've added the changes based on this comment. Thank you @NataliaTepluhina

Reply:
Thank you for the changes, @404answernotfound! Looks great to me, let's get it merged 🎉